SmartDeviceLink/public/SDLVideoStreamingCapability.h

```diff
@@ -13,6 +13,7 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+#define DEFAULT_FRAME_RATE 15
```

Comment:
Can you revert this back to the way you had it in the first review? I think it makes sense for the `SDLStreamingVideoLifecycleManager` to manage the logic for setting the default frame rate value if `preferredFPS` is `nil`.

Reply:
Understood. Updated accordingly.